public/app/app.ts

```diff
@@ -166,6 +166,9 @@ function initEchoSrv() {
 
   window.addEventListener('load', (e) => {
     const loadMetricName = 'frontend_boot_load_time_seconds';
+    // should match names in public/views/index-template.html
+    const jsLoadMetricName = 'frontend_boot_js_done_time_seconds';
+    const cssLoadMetricName = 'frontend_boot_css_time_seconds';
 
     if (performance && performance.getEntriesByType) {
```

Comment:
This is bit confusing. To actually measure anything we have to explicitly report it but also backend needs to know about the measurement. So gathering it dynamically like this does not make sense to me. If browser starts reporting some more metrics we wouldn't automatically add it to prometheus because the histogram isn't registered. So I think it would make sense to just statically write down the metrics here for the paints and comment that it needs to match the `frontendmetrics.go` registered histograms.

Reply:
It measures the time up until that line is executed - from initial page load to window load event.

Reply:
Makes sense. I'll change them to be statically defined in frontend.